### DISCUSSION TITLE
How to contribute to Cruiser?

### DISCUSSION THREAD

**Discussion by imagingbook** [devemux86/cruiser#463] [Q&A] [answered]
@devemux86 Thanks for this great application! Is the `Cruiser` project open-source and is there any way to contribute?

**Comment by devemux86** [comment, marked as answer]
Hi @imagingbook and thanks for the kind words!

I am thinking to open source Cruiser in the future.

If you like Cruiser, please tell others about it! 💕 💕

Cruiser subscriptions is the best way to support its development. :slightly_smiling_face: 

New features are added constantly:
- https://github.com/devemux86/cruiser/discussions/451
